### PR TITLE
T256 - send email to hosts who didn't complete registration

### PR DESCRIPTION
**Pull request by opinyo**

### CODE REVIEW

**Comment by opinyo** [comment]
[triggered email למשתמשים שלא סיימו הרשמה](https://trello.com/c/yTnWdBNG/256-triggered-email-%D7%9C%D7%9E%D7%A9%D7%AA%D7%9E%D7%A9%D7%99%D7%9D-%D7%A9%D7%9C%D7%90-%D7%A1%D7%99%D7%99%D7%9E%D7%95-%D7%94%D7%A8%D7%A9%D7%9E%D7%94)

File: app/mailers/mail_preview.rb [hunk 1]
```diff
@@ -87,4 +87,8 @@ def witness_assigned_therapist_he
     HostMailer.witness_assigned(host_id, witness_id, :he)
   end
 
-end
\ No newline at end of file
+  def incomplete_registration_he
```
Comment (aryehbeitz) [inline app/mailers/mail_preview.rb]: remove this

Comment (opinyo) [reply app/mailers/mail_preview.rb]: Why?

Comment (aryehbeitz) [reply app/mailers/mail_preview.rb]: @opinyo 
1. it's not used anywhere in the code
2. it's not good practice to take the last id
3. does't incomplete_registration already do the job?
to be clear, i am asking to remove the method `incomplete_registration_he`

Comment (opinyo) [reply app/mailers/mail_preview.rb]: It's supposed to be like that in mail preview:
https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

Comment (aryehbeitz) [reply app/mailers/mail_preview.rb]: @opinyo what's the link for previewing?

Comment (opinyo) [reply app/mailers/mail_preview.rb]: @aryehbeitz  /mail_view

**Pull request merged**